components/picker/stories/picker.stories.js

```diff
@@ -141,113 +198,411 @@ export default {
 			}
 		},
 	},
+	decorators: [
+		withDownStateDimensionCapture(".spectrum-Picker:not(:disabled, .is-disabled, .is-loading)"),
+		(Story, context) => {
+			if (!window.isChromatic()) return Story(context);
+			return html`
+				<style>
+					.spectrum-Detail { display: inline-block; }
+					.spectrum-Typography > div {
+						border: 1px solid var(--spectrum-gray-200);
+						border-radius: 4px;
+						padding: 0 1em 1em;
+						/* Why seafoam? Because it separates it from the component styles. */
+						--mod-detail-font-color: var(--spectrum-seafoam-900);
+					}
+				</style>
+				<div
+					style=${styleMap({
+						display: "flex",
+						flexDirection: "column",
+						alignItems: "flex-start",
+						gap: "1rem",
+						"--mod-detail-margin-end": ".3rem",
+					})}
+				>
+					${Story(context)}
+				</div>
+			`;
+		},
+	],
 };
 
+/**
+ * Group of Pickers for default, hover, open + hover, open, keyboard focus, and disabled.
+ */
+const States = ({
+	fieldLabelPrefix = "picker",
+	...args 
+}) => html`
+	<div>
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Default"],
+		})}
+		${Template({
+			...args,
+			fieldLabelId: fieldLabelPrefix + "-default",
+		})}
+	</div>
+	<div>
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Hover"],
+		})}
+		${Template({
+			...args,
+			isHovered: true,
+			fieldLabelId: fieldLabelPrefix + "-hover",
+		})}
+	</div>
+	<div>
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Active (down state)"],
+		})}
+		${Template({
+			...args,
+			isActive: true,
+			fieldLabelId: fieldLabelPrefix + "-active",
+		})}
+	</div>
+	<div>
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Keyboard Focused"],
+		})}
+		${Template({
+			...args,
+			isKeyboardFocused: true,
+			fieldLabelId: fieldLabelPrefix + "-key-focused",
+		})}
+	</div>
+	<div>
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Disabled"],
+		})}
+		${Template({
+			...args,
+			isDisabled: true,
+			fieldLabelId: fieldLabelPrefix + "-disabled",
+		})}
+	</div>
+	<div>
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Loading"],
+		})}
+		${Template({
+			...args,
+			isLoading: true,
+			fieldLabelId: fieldLabelPrefix + "-loading",
+		})}
+	</div>
+	<div style="padding-bottom: 130px;">
+		${Typography({
+			semantics: "detail",
+			size: "s",
+			content: ["Open + Hover"],
+		})}
+		${Template({
+			...args,
+			isHovered: true,
+			isOpen: true,
+			fieldLabelId: fieldLabelPrefix + "-open-hover",
```

Comment:
This was for having unique IDs with these form fields, to avoid some of the accessibility warnings for form fields with duplicate IDs (in Chrome and the Storybook "Accessibility" tab). And making sure the IDs for the labels match up with the form fields.